resources/reps-platform.make

```diff
@@ -8,15 +8,34 @@ core = 7.x
 projects[admin_language][subdir] = "contrib"
 projects[admin_language][version] = 1.0-beta1
 
+projects[admin_theme][subdir] = "contrib"
+projects[admin_theme][version] = 1.0
+
+projects[administration_language_negotiation][subdir] = "contrib"
```

Comment:
I don't see any proof that someone from the QA team approved this contrib and I am not saying this contrib is not OK...